routes/web.php

```diff
@@ -226,6 +226,7 @@
         Route::get('get-state/{state}', 'Payment\TaxController@getState');
         Route::get('get-tax', ['as' => 'get-tax', 'uses' => 'Payment\TaxController@getTax']);
         Route::get('get-loginstate/{state}', 'Auth\AuthController@getState');
+        Route::get('get-loginstate/{state}', 'Auth\AuthController@getState');
```

Comment:
what's this?

Reply:
Removed. Probably occured while rebasing